### DISCUSSION TITLE
Math fonts are rendered bolder than text fonts

### DISCUSSION THREAD

**Discussion by marcin-serwin** [typst/typst#2982] [General]
### Description

When math and text fonts are set to use matching fonts, the math glyphs are rendered as slightly bolder. For example, the following code:
```typst
#set page(margin: 0pt, width: auto, height: auto)

#set text(font: "Latin Modern Roman")
#show math.equation: set text(font: "Latin Modern Math")

1$1$
```
produces this image, in which the second `1` is noticeably bolder:
![image](https://github.com/typst/typst/assets/12128106/66524587-6505-4a1f-906c-c8c9b03014e9)

Compare this to the latex output (using `lualatex`) of:
```latex
\documentclass{standalone}

\usepackage{unicode-math}

\setmainfont{Latin Modern Roman}
\setmathfont{Latin Modern Math}

\begin{document}
1$1$
\end{document}
```
Here both `1`s are identical
![image](https://github.com/typst/typst/assets/12128106/a8639331-903b-4488-a775-65f887886804)

Typst version: `0.10.0 (70ca0d25)`

### Operating system

Linux

### Typst version

- [X] I am using the latest version of Typst

**Comment by Enivex** [comment]
They're likely just defaulting to different font weights (which you can adjust).

Regardless, you should be using New Computer Modern instead of Latin Modern. It has significant improvements.

Alternatively it's a case of https://github.com/typst/typst/issues/576 . In either case I'll convert this to a discussion.

**Reply by marcin-serwin** [reply]
You're right, it's a duplicate of this issue.
